MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,489 @@
+                new SfSegmentItem() {Text="Week", SelectedSegmentTextColor = Colors.Blue},
+                new SfSegmentItem() {Text="Month", SelectedSegmentTextColor = Colors.SkyBlue},
+                new SfSegmentItem() {Text="Year", SelectedSegmentTextColor = Colors.DarkBlue},
+            };
+        segmentedControl.ItemsSource = itemsCollection;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize Selected Segment Border Color
+You can customize the selected segment border color, by using `Stroke` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                Stroke="Green"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            Stroke = Colors.Green,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize Selected Segment Border Thickness
+You can customize the selected segment border thickness, by using `StrokeThickness` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                StrokeThickness="1"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            StrokeThickness = 1,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Notifying Segment Selection Changes
+The `SelectionChanged` event is triggered, once the segment is selected in the Segmented control. The `SelectionChangedEventArgs` has the following values, which provides information for `SelectionChanged` event:
```

Comment:
The "SelectionChanged" event is triggered once the segment is selected in the segmented control. The "SelectionChangedEventArgs" has the following values, which provide information for the "SelectionChanged" event.

Reply:
content changed